tests/results/83e68ee10431fb8c25185a308a532e826b5a7c85250a7c3dbfb805d2479d3abb_abb_unsec/result.json

```diff
@@ -17,10 +17,6 @@
           [
             "requested_url_ip",
             "155.138.224.36"
-          ],
-          [
-            "actual_url_ip",
-            "155.138.224.36"
```

Comment:
Why did we lose the actual_url_ip for that one, and not for others like slashdot?

Reply:
I'm not sure that we need to change Kangooroo, but this test was generated when it was answering.
I think we can manually edit the kangooroo result.json so that it mimic what the behaviour would be, even if it's not a live website anymore.